core_contracts/dividends/dividends.py

```diff
@@ -489,15 +533,17 @@ def tokenFallback(self, _from: Address, _value: int, _data: bytes) -> None:
         self._check_for_new_day()
         day = self._snapshot_id.get()
         self._daily_fees[day][str(self.msg.sender)] += _value
-        self.DividendsReceived(_value, f"{_value} tokens received as dividends token: {self.msg.sender}")
+        self.DividendsReceived(_value, f"{_value} tokens received as dividends token: {self.msg.sender} "
+                                       f"Snapshot id: {self._snapshot_id.get()}")
         self._amount_received_status.set(True)
 
     @payable
     def fallback(self):
         self._check_for_new_day()
         day = self._snapshot_id.get()
         self._daily_fees[day][str(ZERO_SCORE_ADDRESS)] += self.msg.value
-        self.DividendsReceived(self.msg.value, f"{self.msg.value} ICX received as dividends")
+        self.DividendsReceived(self.msg.value, f"{self.msg.value} ICX received as dividends. "
```

Comment:
Let's make the snapshot id the indexed datum for the DividendsReceived eventlog.

Reply:
okay, will add a new version of event log that would index the snapshot id.

Reply:
Has been addressed in https://github.com/balancednetwork/balanced-contracts/pull/256/commits/0e28d6e7992c67df6d5b0efe254ab3059b070fff